docs/content/content/types.md

```diff
@@ -60,6 +60,9 @@ Create a directory with the name of the type in `/layouts`. Type is always singu
 ### Create single template
 Create a file called `single.html` inside your directory. *E.g. `/layouts/post/single.html`*.
 
+### Create list template
+Create a file called `post.html` inside your directory. *E.g. `/layouts/section/post.html`*.
```

Comment:
@moorereason what do you mean? According to the docs [here] (https://gohugo.io/templates/list/) it should be `/layouts/section/YOURSECTION.html`. Should I add something else or change it to be more clear?